### DISCUSSION TITLE
Please don't force this feed on me !!

### DISCUSSION THREAD

**Discussion by kha7iq** [community/community#53970] [Other Feature Feedback, Questions, & Ideas] [answered]
### Product Feedback

This new feed is terrible, i don't want to scroll through the whole page to see some random guy started some repository which i never signed up for, nor am i interested in such a thing.
There are already options like "Trending" if i want to see  whats gaining traction or which software people find useful. If i want to see activity from specific users or organizations i will follow them we already have this option.
Don't know why i am seeing all this garbage and i can not find an option to disable this.

**Comment by AnnaRusu** [comment, marked as answer]
Just use the **Following** tab instead of **For you**.

**Reply by non-descriptive** [reply]
Is there a way to make it default?

**Reply by AnnaRusu** [reply]
For now, from what I know, the only solution is to select it. It will be selected by default while you don't erase your cookies. 

Unfortunately it's just a cookies stuff, not a user setting.

**Reply by kha7iq** [reply]
I don't think they ever going to fix anything users provide feedback for, they do it the MS way. 
Accepting this answer as this is the only way till they also make "For you" default and remove the "Following"  tab for good.

**Reply by mewmew** [reply]
> For now, from what I know, the only solution is to select it. It will be selected by default while you don't erase your cookies.
> 
> Unfortunately it's just a cookies stuff, not a user setting.

Hi Anna,

Thanks for the info.

I clear my cookies on a regular basis. Is there any chance of making this choice recorded in between sessions? I would like to make the "Following" default, as I find it more useful than the "For you" tab.

Kind regards,
Robin
